portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase

```diff
@@ -1194,32 +1194,6 @@ definition {
 			locator1 = "TextInput#ANY");
 	}
 
-	@description = "LPS-166479. Created (theme SVG extensions), the theme SVG extensions admin page lists the name and type"
-	@priority = 4
-	test ThemeSvgTypeCanBeCreated {
-		property custom.properties = "feature.flag.LPS-166479=true";
```

Comment:
Here we would like to keep the test but remove the declaration of the additional property, so I think (@CarlosBrichete please confirm) that removing only line 1200 is enough. The rest of the test has to remain

Reply:
Yes, if we are removing the FF we can just get rid off that property and the test should keep working as before (not only that, the test should keep working removing the property or keeping it, it doesn't matter)

Reply:
It's seems that custom.properties is not being used, so looks safe to remove it, doesn't it? but...for what it was been used when FF was enabled? (just curiosity) @dsanz @CarlosBrichete

Reply:
custom.properties is for the portal-ext.properties file but in CI. When the FF is removed from the code and you still keep the property in the test, it just does nothing, so it won't break the test